docker-assets/entrypoint

```diff
@@ -4,10 +4,38 @@ function urlescape() {
   PAYLOAD="$1" ruby -rcgi -e "puts CGI.escape(ENV['PAYLOAD'])"
 }
 
+if [[ -z $ACG_CONFIG ]]; then
+  DATABASE_NAME="sources_production"
+  export WEB_PORT=3000
+else
+  CONFIG_STRING=`ruby<<EOF
+  require 'json'
+  file = File.read(ENV['ACG_CONFIG'])
+  data = JSON.parse(file)
+  puts data['database']['hostname']
+  puts data['database']['port']
+  puts data['database']['username']
+  puts data['database']['password']
```

Comment:
I'm not sure we should be `put`'ing the password to STDOUT.

Reply:
Any idea how to parse the json file better in shell?

Reply:
Oh. I missed that it's just redirecting. I apologize.